hq/app/schedule/Dynamo.scala

```diff
@@ -42,6 +42,7 @@ class Dynamo(client: AmazonDynamoDB, tableName: Option[String]) extends Attribut
       val alerts = r("alerts").getL.asScala.map { a =>
         val alertMap = a.getM.asScala
         IamAuditAlert(
+          IamAuditNotificationType.fromName(alertMap("type").getS),
```

Comment:
I wonder what the dynamoDB table will start to look like once we turn on the IAM job and this new job? Will it be too busy? Will it be easy to understand? Do we need a CODE table as well as a PROD one for testing things? This PR looks great, so let's merge this, but perhaps we can have this discussion later.

Reply:
Yeah, happy to talk about that soon. CODE table sounds useful on the face of it but we don't really have the concept of CODE IAM users which makes it harder to isolate a CODE job/table. Which I suppose is why you didn't set up a CODE table in the first place. Something to think about!